doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -237,6 +237,12 @@ nRF Desktop
   * A debug configuration enabling the `Fast Pair`_ feature on the nRF54L15 PDK with the ``nrf54l15pdk/nrf54l15/cpuapp`` board target.
   * An application versioning using the :file:`VERSION` file.
     The versioning is only applied to the application configurations that use the MCUboot bootloader.
+  * The :ref:`CONFIG_DESKTOP_USB_HID_REPORT_SENT_ON_SOF <config_desktop_app_options>` Kconfig option to :ref:`nrf_desktop_usb_state`.
```

Comment:
:ref:`nrf_desktop_usb_state` module.

Reply:
That would result in duplicated `module` word. Not applied